station_data.py

```diff
@@ -4,8 +4,21 @@
 from StringIO import StringIO
 from numpy import genfromtxt
 import ast
+import re
```

Comment:
Fair enough :-) the other pull requests uses requests, which can convert a response with JSON to a python dict (more or less like r = requests.get('').json())